controlplane/kubeadm/internal/controllers/controller.go

```diff
@@ -672,7 +672,8 @@ func (r *KubeadmControlPlaneReconciler) reconcileDelete(ctx context.Context, con
 			continue
 		}
 
-		log.Info("Deleting control plane Machine")
+		log.WithValues(controlPlane.StatusToLogKeyAndValues(nil, machineToDelete)...).
+			Info("Deleting Machine (KCP deleted)")
```

Comment:
```suggestion
			Info("Deleting Machine (KubeadmControlPlane deleted)")
```
That's what we did for condition messages at least